tests/func-tests/monitoring_test.go

```diff
@@ -66,22 +108,73 @@ func checkRunbookUrls(client kubecli.KubevirtClient) {
 		Timeout(10 * time.Second).
 		Do(context.TODO()).Into(&prometheusRule)
 	ExpectWithOffset(1, err).ShouldNot(HaveOccurred())
-
-	for _, group := range prometheusRule.Spec.Groups {
-		for _, rule := range group.Rules {
-			if len(rule.Alert) > 0 {
-				ExpectWithOffset(1, rule.Annotations).ToNot(BeNil())
-				url, ok := rule.Annotations["runbook_url"]
-				ExpectWithOffset(1, ok).To(BeTrue())
-				checkAvailabilityOfUrl(url)
-			}
-		}
-	}
-
+	return prometheusRule
 }
 
 func checkAvailabilityOfUrl(url string) {
 	resp, err := http.Get(url)
 	ExpectWithOffset(2, err).ToNot(HaveOccurred())
 	ExpectWithOffset(2, resp.StatusCode).Should(Equal(200))
 }
```

Comment:
Two comments about this method:

1. Use Head instead of Get
2. Use timeout

For example:
```suggestion
// at the top of the file, inside the `Describe`:
var runbookClient := http.DefaultClient
runbookClient.Timeout = time.Second * 3
    
func checkAvailabilityOfUrl(url string) {
	resp, err := runbookClient.Head(url)
	ExpectWithOffset(2, err).ToNot(HaveOccurred())
	ExpectWithOffset(2, resp.StatusCode).Should(Equal(200))
}
```